alpha/apps/kaltura/lib/myPlaylistUtils.class.php

```diff
@@ -327,34 +328,25 @@ public static function getStaticPlaylistFilters(entry $playlist)
 		return array($filter);
 	}
 	
-	public static function executeStaticPlaylist ( entry $playlist , $filter  = null, $detailed = true, $pager = null )
-	{
-		$entry_id_list_str = $playlist->getDataContent();
-		if(kEntitlementUtils::getEntitlementEnforcement() &&
-			kCurrentContext::$ks_object &&
-			kCurrentContext::$ks_object->getDisableEntitlementForPlaylistPlaylistId() === $playlist->getEntryId())
-		{
-			kEntitlementUtils::initEntitlementEnforcement(null, false);
-			entryPeer::setDefaultCriteriaFilter();
-			$result =  self::executeStaticPlaylistFromEntryIdsString($entry_id_list_str, $filter, $detailed, $pager);
-			kEntitlementUtils::initEntitlementEnforcement();
-			entryPeer::setDefaultCriteriaFilter();
-		}
-		else
-		{
-			$result = self::executeStaticPlaylistFromEntryIdsString($entry_id_list_str, $filter, $detailed, $pager);
-		}
-
-		return $result;
-	}
-	
-	public static function executeStaticPlaylistFromEntryIdsString($entry_id_list_str, $filter = null, $detailed = true, $pager = null)
+	public static function executeStaticPlaylistFromEntryIdsString($entry_id_list_str, $filter = null, $detailed = true, $pager = null, $playlistId = null)
 	{
 		$entry_id_list = self::getEntryIdsFromStaticPlaylistString($entry_id_list_str);
```

Comment:
Use the vars naming convention here too for _entry_id_list_str_, _entry_id_list._